test/e2e/protractor.conf.js

```diff
@@ -18,21 +18,21 @@
 
```

Comment:
One small request: could we rename this directory from "test" to "tests"? (Since there are, presumably, multiple tests.)

Reply:
Done